motoman_driver/launch/io_relay.launch

```diff
@@ -0,0 +1,12 @@
+<!-- 
+  Wrapper launch file for I/O relay node.
+-->
+<launch>
+  <!-- IP of robot (or PC running simulation) -->
+  <arg name="robot_ip" doc="IP of controller" />
+
+  <!-- put them on the parameter server -->
+  <param name="robot_ip_address" type="str" value="$(arg robot_ip)" />
```

Comment:
I would change this parameter to `robot_ip` to be consistent with the argument

Reply:
This would be a similar change to the private vs global parameters. Probably better to do this in a follow-up PR.